### DISCUSSION TITLE
Include Printer Profile Name in Email/Text

### DISCUSSION THREAD

**Discussion by Shoey5** [berrystephenw/OctoText#100] [Ideas]
Sorry if this is already in there but I haven't been able to find it. 
If you have multiple printers printing, it would be helpful to know which printer is reporting information. 

Instead of using 
OctoText: Printer ERROR!

Add an option to replace OctoText  with the Printer Profile Name in the Subject. Example
MK3:  Printer ERROR!
CR10:  Printer ERROR!

**Comment by berrystephenw** [comment]
Hi Shoey, if you change the appearance name octotext will report that name in the texts or emails.

**Reply by Shoey5** [reply]
Thanks much, that did it!
